src/test/resources/environment.yaml

```diff
@@ -3,6 +3,7 @@ version: 1
 # You only need to configure this, if you plan on white listing proxy boxes and will need to jump
 # through them to talk to the instances that will live in the Cerberus VPC
 # If your not using a proxy either delete this section or make type = DIRECT for direct connecting
```

Comment:
why type = DIRECT instead of just not supplying proxy-config?  

I know this isn't part of this PR

Reply:
If I re-call it was nice when going back and forward from home and work network, you could just change the string temp.

However, I don't think this section is even needed anymore, with the new architecture
It can probably be removed.